apps/server/src/server.module.ts

```diff
@@ -33,8 +34,12 @@ const userEntities = [User, Role, Account];
 		AuthModule,
 		TaskModule,
 		NewsModule,
-		// MailModule, // TODO enable
 		UserModule,
+		MailModule.forRoot({
+			uri: Configuration.get('RABBITMQ_URI') as string,
```

Comment:
why not having all configuration values in use by the module taken from a constants.ts or configuration.ts which gives an overview about what configures/constants the module is using?